tests/mol-tts_multi/EPUB/package.opf

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package xmlns="http://www.idpf.org/2007/opf" xmlns:epub="http://www.idpf.org/2007/ops" version="3.0" xml:lang="en" unique-identifier="pub-id">
+  <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
+    <dc:coverage>Media Overlays</dc:coverage>
+    <dc:creator>Ivan Herman</dc:creator>
+    <dc:date>2022-02-17</dc:date>
+    <dc:description>Tests whether the Reading System implements Text-to-Speech as a fallback when no audio content is provided. Severak fragments are defined in the SMIL file.</dc:description>
+    <dc:identifier id="pub-id">mol-tts_multi</dc:identifier>
+    <dc:language>en</dc:language>
+    <dc:title>Multiple text-to-speech fallback for MO</dc:title>
+    <link rel="dcterms:rights" href="https://www.w3.org/Consortium/Legal/2015/copyright-software-and-document"/>
+    <link rel="dcterms:rightsHolder" href="https://www.w3.org"/>
+    <meta property="belongs-to-collection">should</meta>
+    <meta property="dcterms:isReferencedBy">https://www.w3.org/TR/epub-rs-33/#sec-text-to-speech</meta>
+    <meta property="dcterms:modified">2022-02-17T00:00:00Z</meta>
+    <meta property="media:duration" refines="#md-smil">00:01:46.35</meta>
+    <meta property="media:duration">00:01:46.35</meta>
```

Comment:
Are these two lines right? The spec says "The implicit duration of the text element (and by inference, of the parent par element) is therefore determined by the execution of the Text-to-Speech engine, and cannot be known at authoring time (factors like speech rate, pauses and other prosody parameters influence the audio output)."

Reply:
Good catch. And I think the spec is a bit unclear on this, so I will have to raise an issue for clarification. The fact is: per https://w3c.github.io/epub-specs/epub33/core/#sec-duration the presence of a `media:duration` is _required_ per spec, and epubcheck complains if it is not present (as it should, per spec). This also means that, in the case of TTS, those values are arbitrary. Which is still all right, but I believe the spec text in https://w3c.github.io/epub-specs/epub33/rs/#sec-text-to-speech.

For the moment, I leave these in the OPF file.